release_foreman_rpms

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+if [ -z "$1" ];then
+  echo "Pass version to release, e.g. ./release_foreman_rpms 2.2"
+  exit 1
+fi
+
+VERSION=$1 ./download_rpms
+VERSION=$1 ./sign_rpms
```

Comment:
My issue here with blindly signing is that for every release I did I always checked if the downloaded RPMs were ones that I actually expected. If you're just going to sign without any checks, what's the signing really worth?

That was the idea behind downloading the RPMs to a directory: you can inspect them there. Then you perform the signing operations. Once done, they're moved so you have an archive.

Reply:
I'm not sure that idea has been captured anywhere if I look at the README or our Foreman procedures. If I am someone following either of those, I would just go step by step with no inspection. And, if I am being honest, I don't know what I would be inspecting the RPMs **for**.

Reply:
I always look at the version numbers and if they make sense. For example, is some change expected? Some nightly version of hammer should not be signed and uploaded.

Reply:
That feels like something we should codify through documentation in our procedure, with the list of things to look for, and where possible code to validate the rules otherwise I don't think we can expect consistency among releasers.